ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -308,7 +308,7 @@ public void startup() throws InstantiationException {
       requests = new AmbryServerRequests(storageManager, networkServer.getRequestResponseChannel(), clusterMap, nodeId,
           registry, metrics, findTokenHelper, notificationSystem, replicationManager, storeKeyFactory, serverConfig,
           diskManagerConfig, storeKeyConverterFactory, statsManager, clusterParticipants.get(0));
-      requestHandlerPool = new RequestHandlerPool(serverConfig.serverRequestHandlerNumOfThreads,
+      requestHandlerPool = new RequestHandlerPool(serverConfig.serverRequestHandlerNumSocketServerThreads,
```

Comment:
nit: We could also use this config in VCR Server too - https://github.com/linkedin/ambry/blob/9c685c5389ff6c07523a0cba89c16781b882a4b7/ambry-cloud/src/main/java/com/github/ambry/cloud/VcrServer.java#L245

Reply:
done